metau-capstone/Schedule.m

```diff
@@ -0,0 +1,27 @@
+//
+//  Schedule.m
+//  metau-capstone
+//
+//  Created by Shirley Zhu on 7/12/22.
+//
+
+#import "Schedule.h"
+#import "PFObject+Subclass.h"
+
+@implementation Schedule
+@dynamic arrayOfLevels;
+@dynamic dayNum;
+
++ (nonnull NSString *)parseClassName {
+    return @"Schedule";
+}
+
++ (void) createDay: ( NSArray * _Nullable )arrayOfLevels withNum: ( NSNumber * _Nullable )dayNum withCompletion: (PFBooleanResultBlock  _Nullable)completion {
+    Schedule *newSched = [Schedule new];
+    newSched.arrayOfLevels = arrayOfLevels;
+    newSched.dayNum = dayNum;
+    
+    [newSched saveInBackgroundWithBlock:completion];
```

Comment:
Technically, this definitely works!

In reality, it's not a whole lot of data so hard coding this mapping is also a very valid direction to go. We don't expect this map to change AFAIK, so using the database as a store for this constant data might be _just a little overkill_.